code/admin/language/en-GB/en-GB.com_api.ini

```diff
@@ -80,8 +80,10 @@ COM_API_KEYS_PER_HOUR="Per hour"
 
 ; Config
 COM_API_CONFIG_LOG_LBL="Log Requests"
+COM_API_CONFIG_ALLOW_CORS_LBL="Allow Cross Origin Requests"
+COM_API_CONFIG_ALLOW_CORS_DESC="Choose which method is allowed for Cross Origin Requests."
 COM_API_CONFIG_CORS_LBL="Allow Cross Origin Requests"
-COM_API_CONFIG_CORS_DESC="List of URLs for which to allow CORS requests. Put one URL per like. Use * to allow CORS reqeust from any domains."
+COM_API_CONFIG_CORS_DESC="List of URLs for which to allow CORS requests. Put comma seperated URL's. Ex. <b>https://www.google.co.in, https://www.google.co.in</b>"

```

Comment:
Change this to 

List of URLs for which to allow CORS requests. Put an asterisk (*) to allow CORS requests from all domains. Alternately put a comma separated list of  URL's. Ex. https://techjoomla.com, http://example.com